packages/slate/src/models/node.js

```diff
@@ -1351,6 +1351,41 @@ class Node {
     return path
   }
 
+  /**
+   * refind the path of node if path is changed
```

Comment:
Can you capitalize this comment and add a period to the end to match the rest of the codebase?